airflow-core/src/airflow/ui/src/queries/useTrigger.ts

```diff
@@ -63,6 +64,33 @@ export const useTrigger = ({ dagId, onSuccessConfirm }: { dagId: string; onSucce
   };
 
   const onError = (_error: unknown) => {
+    // Check if error is a 403 (Forbidden) status
+    if (isHttpError(_error, 403)) {
+      toaster.create({
+        description: translate(
+          "triggerDag.toaster.error.forbidden.description",
+          "You don't have permission to trigger this DAG",
+        ),
+        title: translate("triggerDag.toaster.error.forbidden.title", "Permission Denied"),
+        type: "error",
+      });
+    } else if (isHttpError(_error, 401)) {
+      toaster.create({
+        description: translate(
+          "triggerDag.toaster.error.unauthorized.description",
+          "Please log in to trigger DAGs",
+        ),
+        title: translate("triggerDag.toaster.error.unauthorized.title", "Authentication Required"),
+        type: "error",
+      });
+    } else {
+      // Generic error handling for other status codes
+      toaster.create({
+        description: translate("triggerDag.toaster.error.generic.description", "Failed to trigger DAG"),
+        title: translate("triggerDag.toaster.error.generic.title", "Error"),
```

Comment:
You need to add those translation keys to the translation files. Similarly to `triggerDag.toaster.success.title`.

For instance if you check `en/components.json`